src/OneExplorer/ConfigObject.ts

```diff
@@ -23,9 +23,10 @@ import {RealPath} from '../Utils/Helpers';
 import {Logger} from '../Utils/Logger';
 
 /**
- * 'Artifact' means includes
- *   - existing files to run ONE config (base model)
- *   - result files after running ONE config (derived models, )
+ * 'Artifact'
+ * The collective term is and inclusive, it includes two types of files:
+ * (1) Pre-existing files to run ONE config (base model)
+ * (2) Result files after running ONE config (derived models, )
```

Comment:
@hyunsik-yoon [The change you suggested](https://github.com/Samsung/ONE-vscode/pull/1059#discussion_r924053250) has been actually applied here. 😄 PTAL.